service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/PbcService.kt

```diff
@@ -53,43 +78,39 @@ class PbcService(
 
     fun getMarkerEscrowBalance(escrowDenom: String = reserveDenom) =
         // id is marker denom, denom is escrow denom
-        grpcClientService.new().markers.getMarkerEscrow(reserveDenom, escrowDenom)?.amount ?: "0"
```

Comment:
got rid of the new() instantiation cause we weren't using anything but the default singleton anyways